src/applications/static-pages/mhv-signin-cta/components/messages/UnverifiedAlert.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import {
+  SERVICE_PROVIDERS,
+  CSP_IDS,
+} from '@department-of-veterans-affairs/platform-user/authentication/constants';
+import { logoutUrl } from '@department-of-veterans-affairs/platform-user/authentication/utilities';
+import { logoutUrlSiS } from '~/platform/utilities/oauth/utilities';
+import CustomAlert from './CustomAlert';
+
+export const headingPrefix = 'Verify your identity';
+
+/**
+ * Alert to show a user that is not verified (LOA1).
+ * @property {*} recordEvent the function to record the event
+ * @property {string} serviceDescription optional description of the service that requires verification
+ * @property {string} signInService the ID of the sign in service
+ */
+const UnverifiedAlert = ({
+  hasSsoe = false,
+  recordEvent = recordEventFn,
+  serviceDescription,
+  signInService = CSP_IDS.ID_ME,
+}) => {
+  const signinServiceLabel = SERVICE_PROVIDERS[signInService]?.label;
+  const headline = serviceDescription
+    ? `${headingPrefix} to ${serviceDescription}`
+    : headingPrefix;
```

Comment:
Suggestion: It would be less confusing if the HTML provided the full headline, or the component provided its own.

Reply:
Not sure what you mean. We have different alerts that have different headlines, but the service description is shared.

Reply:
I looked into this more. The problem is that the headline is set by the template provided by CAIA and the only thing that can change in the headline is the service description. Here are the templates: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/content-patterns-and-standards/sign-in-alerts.md#verify-alert-for-loa1ial1-logingov-and-idme-accounts.